concourse/pipelines/create-microbosh.yml

```diff
@@ -225,7 +240,8 @@ jobs:
 
             ./build_manifest.sh \
               ../../../bosh-secrets/bosh-secrets.yml \
-              ../../../terraform-outputs/terraform-outputs.yml \
+              ../../../terraform-outputs/bosh.terraform-outputs.yml \
```

Comment:
I was about to do it, but there are a lot of things to change. First the script is different in both cases (one uses args, other gets vars) and then I find that other things need refactoring (e.g. `inputs` and `outputs` should be before `run`, etc).

Also, the different pipelines use different "patterns" to solve the same problems.

I would rather suggest that we do a PR to clean up and homogenise the pipelines, and do not change it here.